### ISSUE TITLE
listen syntax unusable in galaxy release, but in examples

### ISSUE THREAD

**Issue by Karunamon** [nginxinc/ansible-role-nginx#168]
I was reading through the examples here while creating my own config, and I see a lot of references to a `port:` variable as a direct child of `nginx_http_template`_ (as opposed to `listen:`). However, this does not appear to work as of the latest git commit 3b0e39f8709173db5bc64f35060894490001e14b, to be replaced with listen blocks.

This led to some mind-bending troubleshooting before I figured out that `listen`: doesn't work in the current release version available on Galaxy (also meaning that `opts` can't be set), and works fine when using the latest commit. Cutting a new release would make the stuff on galaxy work fine.

----

**tl;dr:** `port` outside of a `listen` should be excised from the examples and a new release cut since the examples on galaxy can't be used directly and options can't be set.

----

Thanks much for your work on this role, you've made my life a lot easier! 😃

**Comment by alessfg** [comment]
The latest release (0.13.0) just got published!
